### ISSUE TITLE
Accessibility issue with form elements in polls

### ISSUE THREAD

**Issue by sidp** [mastodon/mastodon#22908]
### Steps to reproduce the problem

1. Create a multiple choice poll
2. View the poll as a different user.
3. Check one or more of the options.

### Expected behaviour

The checked options should be distinguishable not only by their color.

### Actual behaviour

It is not clear which options are checked.

### Detailed description

Only the use of color can make it difficult to use the interface for some users. This is a [WCAG level A criteria](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html).

Example:

<img width="381" alt="image" src="https://user-images.githubusercontent.com/726932/210269447-e0ccbea3-5234-416e-bf24-5825301b8eea.png">

This issue could be resolved by adding a checkmark symbol in the active checkboxes to make it clearer which ones are checked.

A similar issue (https://github.com/mastodon/mastodon/issues/20721) was previously filed about the sensitive media checkbox in the compose UI.

### Specifications

Accessibility issue present in all browsers.
